### ISSUE TITLE
Note on Raspberry Pi 3 Bluetooth Audio

### ISSUE THREAD

**Issue by CloneMMDDCVII** [MycroftAI/documentation#18]
A few months ago, I tried to run Picroft in a machine that I could control via a bluetooth headset.
After multiple frustrating attempts, I stumbled upon the reason why no configuration or OS permitted me to do that : the Raspi 3's bluetooth drivers do not support audio input.

Just thought a note of this could be made here : https://github.com/mycroftai/docs-rewrite/blob/master/_pages/picroft-audio.md to avoid haing other people try the same thing I did #

**Comment by CloneMMDDCVII** [comment]
Note that (despite testing should be made to verify this) I remember it to work using an external bt dongle.

**Comment by KathyReid** [comment]
Great suggestion @CloneMMDDCVII, thank you very much for raising this issues. 
Will flag this for update in the documentation.

**Comment by krisgesling** [comment]
I've added a warning to the Picroft page and linked to a more recent Forum thread on the topic. 

TLDR; not currently supported, please contribute if you have a magic answer :)
